runtime/src/main/java/org/corfudb/runtime/clients/ClientResponseHandler.java

```diff
@@ -84,11 +85,18 @@ public boolean handle(ResponseMsg response, ChannelHandlerContext ctx) {
                     // For errors, we always want to completeExceptionally
                     errorHandlerMap.get(errorCase).handle(response, ctx, router);
                 } catch (Throwable e) {
+                    log.warn("Server threw exception for {} with request_id: {}",
+                            response.getPayload().getPayloadCase(),
+                            response.getHeader().getRequestId(),
+                            e);
```

Comment:
Is this usage typically reserved for `log.error`?

Reply:
I set to `warn` since previously in the `BaseHandler` there was:
```
    /**
     * Generic handler for a server exception.
     */
    @ClientHandler(type = CorfuMsgType.ERROR_SERVER_EXCEPTION)
    @Deprecated
    private static Object handleServerException(CorfuPayloadMsg<ExceptionMsg> msg,
                                                ChannelHandlerContext ctx, IClientRouter r) throws Throwable {
        log.warn("Server threw exception for request {}", msg.getRequestID(),
                msg.getPayload().getThrowable());
        throw msg.getPayload().getThrowable();
    }
```
I'm a little inclined to use `warn` since sometimes the server throws an exception doesn't necessarily mean there is an error?

Reply:
Sorry, i was referring to the use of the parameter `e`.

Reply:
Removed, thanks!